etabotsite/etabotapp/views.py

```diff
@@ -6,6 +6,17 @@
 from .models import Project
 from .models import TMS
 from .permissions import IsOwner
+from django.views.decorators.csrf import ensure_csrf_cookie
```

Comment:
Is the import used anywhere in the file? If not, please remove.

Reply:
yes, using it for index view